### PR TITLE
feat: about markdown swtich to i18n  translate

### PR DESCRIPTION
**Pull request by eepson123tw**
I would like to explain that due to regional differences, word choices may vary. 
Therefore, I have initially implemented this hard-coded approach to accommodate these variations.

### CODE REVIEW

File: app/pages/about.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import enAbout from '@/static/content/en/about.md'
+import jpAbout from '@/static/content/jp/about.md'
+import zhHansAbout from '@/static/content/zh-Hans/about.md'
+import zhHantAbout from '@/static/content/zh-Hant/about.md'
+
+const layout = 'content'
```
Comment (KazariEX) [inline app/pages/about.vue:7]:
```suggestion
definePageMeta({
  layout: 'content'
})
```

Comment (eepson123tw) [reply app/pages/about.vue]: OK THX~

**Comment by antfu** [comment]
I would suggest to follow this convention to support them and make it easier to scale and support more languages in the future: https://github.com/maru-re/maru/blob/ca45e66a964484c0785265838328d0bbf392d6ce/app/components/qifi/QifiHowToScan.ts

**Comment by eepson123tw** [comment]
Wow I miss this out,will change to this.

**Pull request merged**